
  # ignore cached value otherwise libwrap is not added to LIBS

--- configure
+++ configure
@@ -24937,7 +24937,7 @@
   # next, we search for the wrap library (libwrap.*)
   { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for libwrap" >&5
 printf %s "checking for libwrap... " >&6; }
-  if test ${sc_cv_have_libwrap+y}
+  if false
 then :
   printf %s "(cached) " >&6
 else $as_nop
